spec/rubocop/cop/layout/rescue_ensure_alignment_spec.rb

```diff
@@ -460,128 +460,138 @@ def foo
+        it 'registers an offense' do
+          expect_offense(<<~RUBY)
+            #{modifier} def test
+              'foo'
+              ensure
+              ^^^^^^ `ensure` at 3, 2 is not aligned with `#{modifier} def test` at 1, 0.
+              'baz'
+            end
+          RUBY
 
-    context 'ensure with def' do
-      it 'registers an offense' do
-        expect_offense(<<~RUBY)
-          private def test
-            'foo'
+          expect_correction(<<~RUBY)
+            #{modifier} def test
+              'foo'
             ensure
-            ^^^^^^ `ensure` at 3, 2 is not aligned with `private def test` at 1, 0.
-            'baz'
-          end
-        RUBY
+              'baz'
+            end
+          RUBY
+        end
 
-        expect_correction(<<~RUBY)
-          private def test
-            'foo'
-          ensure
-            'baz'
-          end
-        RUBY
+        it 'correct alignment' do
+          expect_no_offenses(<<~RUBY)
+            #{modifier} def test
+              'foo'
+            ensure
+              'baz'
+            end
+          RUBY
+        end
       end
 
-      it 'correct alignment' do
-        expect_no_offenses(<<~RUBY)
-          private def test
-            'foo'
-          ensure
-            'baz'
-          end
-        RUBY
-      end
-    end
+      context 'ensure with defs' do
+        it 'registers an offense' do
+          expect_offense(<<~RUBY)
+            #{modifier} def Test.test
+              'foo'
+              ensure
+              ^^^^^^ `ensure` at 3, 2 is not aligned with `#{modifier} def Test.test` at 1, 0.
+              'baz'
+            end
+          RUBY
 
-    context 'ensure with defs' do
-      it 'registers an offense' do
-        expect_offense(<<~RUBY)
-          private def Test.test
-            'foo'
+          expect_correction(<<~RUBY)
+            #{modifier} def Test.test
+              'foo'
             ensure
-            ^^^^^^ `ensure` at 3, 2 is not aligned with `private def Test.test` at 1, 0.
-            'baz'
-          end
-        RUBY
+              'baz'
+            end
+          RUBY
+        end
 
-        expect_correction(<<~RUBY)
-          private def Test.test
-            'foo'
-          ensure
-            'baz'
-          end
-        RUBY
+        it 'correct alignment' do
+          expect_no_offenses(<<~RUBY)
+            #{modifier} def Test.test
+              'foo'
+            ensure
+              'baz'
+            end
+          RUBY
+        end
       end
+    end
 
-      it 'correct alignment' do
-        expect_no_offenses(<<~RUBY)
-          private def Test.test
-            'foo'
-          ensure
-            'baz'
-          end
-        RUBY
-      end
+    context 'with private modifier' do
+      include_examples 'access modifier', 'private'
+    end
+
+    context 'with private_class_method modifier' do
+      include_examples 'access modifier', 'private_class_method'
     end
```

Comment:
This expected behavior may be more clearer by writing a spec of `public_class_method`.